service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -76,6 +88,10 @@ public ResponseEntity<PostMethodResponse> postMethod(PostMethodRequest postMetho
     try {
       workflowDescription = cromwellService.describeWorkflow(postMethodRequest.getMethodUrl());
 
+      List<WorkflowInputDefinition> inputs = womToCbasInputBuilder(workflowDescription);
+
+      List<WorkflowOutputDefinition> outputs = womtoolToCbasOutputs(workflowDescription);
```

Comment:
Small nit: It might be more readable to keep the function names for inputs and outputs similar. For example name of  both functions can be `womtoolToCbasInputs` and `womtoolToCbasOutputs` or they can be `womToCbasInputBuilder` and `womToCbasOutputBuilder`.